.github/workflows/main.yml

```diff
@@ -64,20 +65,18 @@ jobs:
         id: meta
         uses: docker/metadata-action@v3
         with:
-          images: |
-            chirpstack/chirpstack-application-server
+          images: chirpstack/${{ github.event.repository.name }}
           tags: |
+            type=ref,event=branch
             type=semver,pattern={{version}}
             type=semver,pattern={{major}}
             type=semver,pattern={{major}}.{{minor}}
-      -
-        name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
       -
         name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
       -
         name: Login to DockerHub
+        if: github.event_name != 'pull_request'
```

Comment:
With the above comment, this could be made more explicit and changed to `startsWith(github.ref, 'refs/tags/v')`. That also makes it more secure as new `event_name` types can be added in the future for which we don't want to perform this step.

Reply:
updated